### DISCUSSION TITLE
Welcome to CommandStation-EX Discussions!

### DISCUSSION THREAD

**Discussion by FrightRisk** [DCC-EX/CommandStation-EX#213] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JamesHofmann37** [comment]
I am trying to connect with Fred Decker. I am retired teacher of Technology, FIRST Robotics coach after 31 years.
Can you help me connect with Fred? my email is: JamesHofmann37@gmail.com
I am a new NMRA member. I attended the Amherst rail show these last two years. 
(973)903-9774 cell
